src/hooks/useLocalization.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * This `useLocalization is only for exportation purposes. It is not used in the project.
+ * It shouldn't be used internally.
+ */
+import { useLocalization } from '../lib/LocalizationContext';
```

Comment:
```suggestion
/**
 * This file is only for re-exporting the `useLocalization` hook.
 * It should not be used internally within the project.
 */
```
Adjusted the comment a bit and removed a duplicated import line. The below line will do the same work.

Reply:
Thanks for adjusting the comment!
I have to import it because of the `export default`!